packages/ag-charts-community/src/chart/flowProportionChart.ts

```diff
@@ -12,6 +12,10 @@ export class FlowProportionChart extends Chart {
     static readonly className = 'FlowProportionChart';
     static readonly type = 'flow-proportion' as const;
 
+    override getChartType() {
+        return 'flow-proportion' as const;
```

Comment:
Could this not on the chart base-class as `return this.constructor.type`?

Reply:
Yes, but let's do it in a follow-up as this is a big change.